### PR TITLE
163479020 operator e2e tests

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
* New e2e test to test creating new transport operator
   
# Fixed
* Select new transport operator as acting operator by default

### CODE REVIEW

File: cypress/integration/operator_spec.js [hunk 1]
```diff
@@ -31,7 +36,68 @@ describe('Operator creation basic tests', function () {
...
+        cy.get('#btn-add-new-transport-operator').click();
+
+        // Business id is already added
+        cy.get('#input-business-id').type('1234567-8');
+        // Check warning
+        cy.contains(testOp1.businessIdExistsWarning);
+
+        // Try creating random business-id
+        cy.get('#input-business-id').clear(); // Clear business-id
+        cy.get('#input-business-id').type(randomBusinessid());
```
Comment (philosotractor) [inline cypress/integration/operator_spec.js]: While which approach to take is a matter of debate, using a random id may lead to test sometimes failing and if it's rerun, it passes. E.g. if random id outputs '1234567-8'. 
Due to human laziness the issue might be overlooked then.
I'd prefer the apprach that a running test should have always the same result, so the valid business id should be hard-coded.
That way it's also possible to cover systematically the different types of id validation, e.g. length/characters/exists/other.

File: cypress/integration/operator_spec.js [hunk 2]
```diff
@@ -31,7 +36,68 @@ describe('Operator creation basic tests', function () {
...
+        cy.wait(500);
+        cy.contains(newOperatorName);
+
+        // Ensure that other pages have it selected by default
+        cy.visit('/#/pre-notices');
+        cy.contains(newOperatorName);
+        cy.visit('/#/routes');
+        cy.contains(newOperatorName);
+
+        // Delete operator
```
Comment (philosotractor) [inline cypress/integration/operator_spec.js]: This should ensure also that deletion works and doesn't produce an error.

File: cypress/integration/operator_spec.js [hunk 3]
```diff
@@ -31,7 +36,68 @@ describe('Operator creation basic tests', function () {
...
+
+        // Add name
+        const newOperatorName = randomName('ctest-op-');
+        cy.get('#input-operator-name').type(newOperatorName);
+
+        // Save
+        cy.get('#btn-operator-save').click();
+
+        // Ensure that operator is correctly added - should be selected by default
+        cy.wait(500);
```
Comment (erno) [inline cypress/integration/operator_spec.js]: Did this break without the wait? Cypress should retry and wait automatically for the .contains to succeed. "Cypress only retries commands that query the DOM: cy.get(), .find(), .contains(), etc" according to the docs.

Comment (rckopteri) [reply cypress/integration/operator_spec.js]: For some reason I could not get cypress to wait for the server response. I did try without the wait(500). But this was the last way to make it work.

File: cypress/integration/operator_spec.js (hunk 2 above)
Comment (rckopteri) [reply cypress/integration/operator_spec.js]: Added

**Pull request merged**